Tests/TemplatesTests/StringsTests.swift

```diff
@@ -29,10 +29,19 @@ class StringsTests: XCTestCase {
         context: context,
         parameters: ["localizeFunction=XCTLocFunc"]
        ),
-       suffix: "-localizeFunction")
+       suffix: "-localizeFunction"),
+      (context: try StencilContext.enrich(context: context, parameters: ["forceFileNameEnum"]),
+       suffix: "-forceFileNameEnum")
     ]
   }
 
+  private lazy var variationsSwift3: VariationGenerator = { name, context in
+    let notSupported = ["-forceFileNameEnum"]
+    return try self.variations(name, context).filter {
+      !notSupported.contains($0.suffix)
+    }
+  }
+
```

Comment:
We actually thought about dropping/removing the Swift 3 templates for the next release… If we decide to do so, such a special case in tests would not be needed anymore – \cc @djbe